doc/setup/install.rst

```diff
@@ -378,6 +456,12 @@ Running Odoo
 
 Once all dependencies are set up, Odoo can be launched by running ``odoo.py``.
 
+.. warning:: For the Enterprise edition, you must specify the :file:`enterprise` addons
```

Comment:
redundant with [this instructions](https://www.odoo.com/documentation/9.0/setup/enterprise.html#on-linux-using-the-source-code). Maybe we should remove it in the other page and put the full here (with a link)